packages/gasket-plugin-docusaurus/README.md

```diff
@@ -36,7 +36,8 @@ launched in your browser.
 
 To be set under `docusaurus` in the `gasket.config.js`.
 
-- `docsDir` - (string) Name of the root documents folder. Default is `.docs`.
+- `rootDir` - (string) Root Docusaurus directory. Default is `.docs`.
+- `docsDir` - (string) Sub-directory for the generated markdown from the docs plugin. Default is `docs`.
```

Comment:
I find these names/definitions hard to follow. What is meant by the root Docusaurus directory? Does that mean the root directory for the documentation website? Is `docsDir` a subdirectory within `rootDir`? Do we need both? I can see why someone might want to customize `rootDir` if that is the root of the generated files, but why would someone need to customize `docsDir`?

Reply:
-  `rootDir` - The root directory of the Docusaurus application which is basically a React application. Yes it's the root directory for the documentation website.
- `docsDir` - The sub directory for the documentation files within the website root - `.md` `.mdx`.

The `rootDir` isn't the _root of the generated files_. The `docsDir` property is essentially the `docsRoot` for the `@gasket/plugin-docs` and that generated directory needs to be inside the Docusaurus website root.

Reply:
Let me just add that both dir names can be optionally changed, and are just made available for custom output locations (probably edge-cases). We've gone back and forth on the names for these. And like @mmason2-godaddy mentioned, although they work together, they serve two purposes: `rootDir` - where does my docusaurus app go, `docsDir` - where does the generated markdown go within my docusaurus app.